lightning/src/ln/channelmanager.rs

```diff
@@ -4960,6 +4960,9 @@ impl<'a, Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>
 						channel.get_cur_counterparty_commitment_transaction_number() > monitor.get_cur_counterparty_commitment_number() ||
 						channel.get_latest_monitor_update_id() < monitor.get_latest_update_id() {
 					// But if the channel is behind of the monitor, close the channel:
+					log_error!(args.logger, "A ChannelManager is stale compared to the current ChannelMonitor! The channel will be closed.");
```

Comment:
IMO, more informative to say "the channel will be closed with the lastest commitment tx extracted from ChannelMonitor"